integration/tests/conftest.py

```diff
@@ -152,11 +152,11 @@ def solana_account(bank_account, pytestconfig: Config, sol_client):
 
 @pytest.fixture(scope="session")
 def erc20_spl(
-    web3_client: NeonChainWeb3Client,
-    faucet,
-    pytestconfig: Config,
-    sol_client,
-    solana_account,
+        web3_client: NeonChainWeb3Client,
```

Comment:
You need install black profile for formatting 
https://medium.com/compendium/automatically-run-black-in-pycharm-on-windows-d2eab855a918

Reply:
Ok, thanks, and I like this approach of setting up black more as it's integrated into pycharm: https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#format-python-code-with-black

Reply:
Done